.svgrrc.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
```

Comment:
Can we add any links to documentation for this file? Impossible to find it on their website.

Reply:
Yeah, the docs are pretty bad.

The default template is here (with no documentation on it's parameters): https://www.smooth-code.com/open-source/svgr/docs/cli/#use-a-specific-template

And the config file docs are here: https://www.smooth-code.com/open-source/svgr/docs/configuration-files/

I'll update with these links.

Reply:
Done